lib/rules/indent.js

```diff
@@ -888,11 +884,15 @@ module.exports = {
                 // alter the expectation of correct indentation. Skip them.
                 // TODO: Add appropriate configuration options for variable
                 // declarations and assignments.
-                if (getVariableDeclaratorNode(node)) {
+                if (getParentNodeByType(node, "VariableDeclarator", "FunctionExpression")) {
+                    return;
+                }
+
+                if (getParentNodeByType(node, "VariableDeclaration", "FunctionExpression")) {
```

Comment:
Do we need to check for VariableDeclaration as well as VariableDeclarator? Please try removing this block and see if tests still pass.